beetmoverscript/docker.d/worker.yml

```diff
@@ -4,7 +4,20 @@ aiohttp_max_connections: 5
 checksums_digests:
   - 'sha512'
   - 'sha256'
-taskcluster_scope_prefix: { "$eval": "TASKCLUSTER_SCOPE_PREFIX" }
+taskcluster_scope_prefixes:
+  $flatten:
+    $match:
+      'COT_PRODUCT == "firefox"':
+        - 'project:releng:beetmover:'
+      'COT_PRODUCT == "thunderbird"':
+        - 'project:comm:thunderbird:releng:beetmover:'
+      'COT_PRODUCT == "mobile"':
+        - 'project:mobile:android-components:releng:beetmover:'
+        - 'project:mobile:focus:releng:beetmover:'
```

Comment:
I assume we also need to add the Fenix corresponding S3 credentials, something like down below, but that's not blocking this PR - I suppose you want to see whether this PR first is backward compatible? :
```
diff --git a/beetmoverscript/docker.d/worker.yml b/beetmoverscript/docker.d/worker.yml
index f06f089..f4e4905 100644
--- a/beetmoverscript/docker.d/worker.yml
+++ b/beetmoverscript/docker.d/worker.yml
@@ -128,6 +128,27 @@ bucket_config:
             nightly_components: 'maven-nightly-stage-s3-upload-bucket'
         dummy: {}
       'COT_PRODUCT == "mobile" && ENV == "prod"':
+        nightly:
+          credentials:
+            id: { "$eval": "FENIX_NIGHTLY_ID" }
+            key: { "$eval": "FENIX_NIGHTLY_KEY" }
+          buckets:
+            thunderbird: 'net-mozaws-prod-delivery-archive'
+          url_prefix: 'https://archive.mozilla.org'
+        release:
+          credentials:
+            id: { "$eval": "FENIX_RELEASE_ID" }
+            key: { "$eval": "FENIX_RELEASE_KEY" }
+          buckets:
+            thunderbird: 'net-mozaws-prod-delivery-archive'
+          url_prefix: 'https://archive.mozilla.org'
+        dep:
+          credentials:
+            id: { "$eval": "FENIX_DEP_ID" }
+            key: { "$eval": "FENIX_DEP_KEY" }
+          buckets:
+            thunderbird: 'net-mozaws-stage-delivery-archive'
+          url_prefix: 'https://ftp.stage.mozaws.net'
         maven-production:
           credentials:
             id: { "$eval": "MAVEN_ID" }
```

Reply:
> I presume we want `project:mobile:fenix:releng:beetmover` as well here? We'll want Focus APKs published too so ++ on adding Focus too from now.

Oops, good catch :) I mean, I *meant* to proactively add Focus.

Reply:
> I assume we also need to add the Fenix corresponding S3 credentials, something like down below, but that's not blocking this PR - I suppose you want to see whether this PR first is backward compatible?

Yeah, I wanted to solve the prefix problem here, and then we can add the bucket creds and such in a followup.

Reply:
Ok, this is on dev-beetmoverscript now, and appears to work. We probably want to add the staging bucket secrets to SOPS and update the worker secrets, and then we can test from staging-fenix. @ahal how much of this do you want to handle? Do you want a chat / writeup / zoom call about this? (@MihaiTabara or I probably need to update SOPS.)

Reply:
Added a follow-up commit with missing changes.

Reply:
I think @hneiva deployed this successfully this week so we should be good here.

Reply:
@hneiva It might be worth checking with @sarah-clements as well whether Focus team wants this uploaded to S3 as well in the future? We don't need to block this PR but if we do, it'd be good to double-check the structure so that when that request comes along, it'll be easier to fill in those credentials.